docs/experimental-module-policy.md

```diff
@@ -0,0 +1,35 @@
+# Drupal Core Experimental module policy
+
+Drupal 8 core introduced the concept of [experimental modules][1].
+These are modules that are provided with Drupal core for testing purposes, but that are not yet fully supported.
+They offer a wide range of functionalities, from migration management
+to site layout management.
+
+Due to the experimental nature of the modules from the OpenEuropa platform,
+we need to define a set of policies on their use inside OpenEuropa Components.
+
+## Minimum stability required
+
+Currently the experimental modules follow the different stability levels defined by Drupal:
+alpha, beta, RC and stable.
+
+In order for OpenEuropa to provide a stable enough set of components, only experimental modules
+in **beta state or greater** will be allowed to be used.
+
+This is done because modules in beta state have a stable API that will only be changed
+in very extreme circumstances. See the official [documentation][2] for more information on the subject.
+
+## Experimental modules in alpha state
+
+Although as a general rule OpenEuropa will not ship with experimental modules in alpha state,
+we recognize the potential value that such modules can provide to our customers.
+
+If from a technical or business view the functionality of an alpha experimental module is absolutely required,
+a component will be allowed to make use of the required alpha experimental module. 
+
+However, in such cases the components will also be labelled as experimental components
+and as such will follow the same life cycle of the experimental module they are using. 
```

Comment:
This is not clear, how do we label our own components as experimental?

Wouldn't it be a better idea to say that any component using any unstable dependencies will remain unstable themselves? So if you use an alpha module your component will also be considered alpha.

I also think that this should not be limited to experimental modules from core, if somebody uses a contrib module or any PHP dependency through Composer that is in alpha state, then their component is also considered alpha.

Reply:
Although I agree with the first part I think the second part radically changes the scope of the ticket. Maybe I'm to picky but shouldn't this be considered separately?

Reply:
Yes you are right, let's discuss whether we can extend this policy also to contrib modules in a follow up. I created OPENEUROPA-568 for this.